tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.TestBot/Microsoft.Bot.Builder.Adapters.Webex.TestBot.csproj

```diff
@@ -4,9 +4,10 @@
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
     <!-- The WebExTeams package isn't signed, so supress the warning. -->
     <NoWarn>$(NoWarn),CS8002</NoWarn>
+    <UserSecretsId>63730bc1-f7d4-4b32-8efe-ce03907b3e0a</UserSecretsId>
```

Comment:
Is the inclusion of this UserSecretsId intentional?

Reply:
Yes, the UserSecretsId is just a GUID that is used to allow you to add config settings to your local copy of the web app. The local settings are stored in your machine under AppData using that GUID and you don't risk accidentally commiting them to git if you modify `appsettings.json` or `launchettings.json`. Having the GUID already in git saves you from having the modify the file to add your own secrets.
For more details see [Safe storage of app secrets in development in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows)

Reply:
Thanks for the explanations Gabo.